common/execpath.cpp

```diff
@@ -71,6 +71,15 @@ std::string find_path(const char *const * env) {
   return ".:/bin:/usr/bin";
 }
 
+std::string find_path(const std::vector<std::string> env) {
+  for (auto& s : env) {
+    if (!strncmp(s.c_str(), "PATH=", 5)) {
```

Comment:
Check out: http://www.cplusplus.com/reference/string/string/compare/
You should use s.compare(0, 5, "PATH=")